dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -163,33 +163,33 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId, userI
 function shareProject() {
   dashboard.project.save(function () {
     var origin = location.protocol + '//' + location.host;
-    var shareUrl = origin + dashboard.project.appToProjectUrl() + '#' + dashboard.project.getCurrentId();
+    var shareUrl = origin + dashboard.project.getPathName();
     var encodedShareUrl = encodeURIComponent(shareUrl);
     var dialog = new Dialog({body: '<img class="modal-image" src="' + appOptions.skin.staticAvatar + '">'
-    + '<div id="project-share" class="modal-content">'
-    + '<p class="dialog-title"><%= I18n.t('project.share_title') %></p>'
-    + '<p><%= I18n.t('project.share_copy_link') %></p>'
-      // TODO: de-dup with apps code once JS common-core work is done.
-    + '<div><input type="text" id="sharing-input" value="' + shareUrl + '" style="cursor: copy; width: 465px;" readonly></div>'
-    + '<button id="continue-button" style="float: right;"><%= I18n.t('project.close') %></button>'
-    + '<div class="social-buttons">'
-    + '  <a href="https://www.facebook.com/sharer/sharer.php?u=' + encodedShareUrl + '" target="_blank" class="popup-window">'
-    + '    <i class="fa fa-facebook"></i>'
-    + '  </a>'
-    + '  <a href="https://twitter.com/intent/tweet?url=' + encodedShareUrl + '&amp;text=Check%20out%20what%20I%20made%20@codeorg&amp;hashtags=HourOfCode&amp;related=codeorg" target="_blank" class="popup-window">'
-    + '    <i class="fa fa-twitter"></i>'
-    + '  </a>'
-    + '  <a id="sharing-phone" href="" onclick="return false;">'
-    + '    <i class="fa fa-mobile-phone" style="font-size: 28pt;"></i>'
-    + '  </a>'
-      // TODO + '  <a href="#" onclick="return false;" id="print"><i class="fa fa-print"></i></a>'
-    + '</div>'
-    + '<div id="send-to-phone" class="sharing" style="display: none;">'
-    + '  <label for="phone">Enter a US phone number:</label>'
-    + '  <input type="text" id="phone" name="phone">'
-    + '  <button id="phone-submit" onclick="return false;">Send</button>'
-    + '  <div id="phone-charges">A text message will be sent via <a href="http://twilio.com">Twilio</a>. Charges may apply to the recipient.</div>'
-    + '</div></div>'
+      + '<div id="project-share" class="modal-content">'
```

Comment:
I think the eventual plan is for this to move to React.  Right now it's awkwardly copy/pasted from `/apps` until this TODO is resolved:

> // TODO: de-dup with apps code once JS common-core work is done.

I.e. there's no good way to include the .ejs template in `/apps` inside of Dashboard.